tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
-  "extends": "@tsconfig/node10/tsconfig.json",
   "compilerOptions": {
-    // required for aggregate-error
     "esModuleInterop": true,
-    "moduleResolution": "node"
+    "module": "esnext",
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es2018"
```

Comment:
I was thinking of migrating all the Octokit repos to use `"extends": "@tsconfig/node10/tsconfig.json` but didn't investigate if that's actually a good idea, given that these target both Node and Browsers. So maybe just keeping them in sync with the settings that work is the better idea

Reply:
Sorry, I arrive late to the discussion here. Maybe I can open a different issue for this. What if we keep extending the @tsconfig/node10 and overwrite the prop/s giving problems with pika instead? Checking the preset seems like most of the props added are the same ones [the preset uses](https://github.com/tsconfig/bases/blob/08bc28bc677531473bc6a7928ce981f506fb5dce/bases/node10.json#L5-L13).

I guess the ones giving problems are these ones?

```json
    "module": "commonjs 🆚 esnext", 
    "skipLibCheck": true 🆚 false,
    "forceConsistentCasingInFileNames": true 🆚 false
```

What do you think @gr2m and @wolfy1339